config/routes.rb

```diff
@@ -8,5 +8,7 @@
     resources :rents, only: %i[index create]
   end
   resources :books, only: %i[index show]
+  get '/book_information', to: 'book_information#show'
```

Comment:
I prefer use `open_library_information` endpoint in books controller. You can check differences between resources, collection and member [in this link](https://carlossanchezperez.wordpress.com/2013/07/21/definir-rutas-en-una-aplicacion-de-rails-con-new-resources-restful-route-customizations/) 
```rb
resources :books, only: %i[index show] do
  collection do
    :open_library_information
  end
end
```